packages/reporter/src/lib/shortcuts.ts

```diff
@@ -23,14 +23,19 @@ class Shortcuts {
         break
       case 's': !appState.isPaused && !appState.studioActive && events.emit('stop')
         break
-      case 'f': events.emit('focus:tests')
+      case 'f': action('toggle:spec:list', () => {
+        appState.toggleSpecList()
+        events.emit('save:state')
+      })()
+
         break
       case 'c': events.emit('resume')
         break
       case 'n': events.emit('next')
         break
       case 'a': action('set:scrolling', () => {
         appState.setAutoScrolling(!appState.autoScrollingEnabled)
+        events.emit('save:state')
```

Comment:
This fixes a bug where this value was not getting saved when the shortcut key was used.